pkg/api/store/cluster/cluster_store.go

```diff
@@ -166,9 +166,28 @@ func (r *Store) Create(apiContext *types.APIContext, schema *types.Schema, data
 		values.PutValue(data, m, managementv3.ClusterFieldAnnotations)
 	}
 
+	setInitialConditions(data)
+
 	return r.Store.Create(apiContext, schema, data)
 }
 
+func setInitialConditions(data map[string]interface{}) {
+	data[managementv3.ClusterStatusFieldConditions] = []map[string]interface{}{
```

Comment:
Can this already exist? A check would be safer so Conditions aren't getting clobbered

Reply:
I will add the check.